### DISCUSSION TITLE
Will all my cram tests still pass when switching to prysk?

### DISCUSSION THREAD

**Discussion by Nicoretti** [prysk/prysk#108] [Q&A] [answered]
Will all my cram tests still pass when switching to prysk?

**Comment by Nicoretti** [comment, marked as answer]
## Current State (Prysk 0.12.1)

### TL;DR
Cram tests should be compatible with prysk, as long as they aren't using cram specific environment variables.
If cram tests do use `cram` specific environment variables please check the `ENV` section bellow.

### ENV
Cram tests which are using cram specific environment variables e.g. `CRAMTMP` need to replace them by the  corresponding `prysk` environment variables.

| cram | prysk |
|-|-|
| CRAM | PRYSK |
| CRAMRC | PRYSKRC |
| CRAMTMP | PRYSK_TEMP |

### :information_source: Info
* AFIK there aren't to many users of prysk as of today, so it is possible that some compatibility issues haven't been discovered yet.
* Long term it is planed to stay compatible with cram's test file format. The environment variables, configuration file(s) and the CLI interface will very likely introduce incompatible changes in the future though.
